openpype/hosts/maya/plugins/publish/extract_gpu_cache.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+from maya import cmds
+
+from openpype.pipeline import publish
+
+
+class ExtractGPUCache(publish.Extractor):
+    """Extract the content of the instance to a GPU cache file."""
+
+    label = "GPU Cache"
+    hosts = ["maya"]
+    families = ["model", "animation", "pointcache"]
+    step = 1.0
+    stepSave = 1
+    optimize = True
+    optimizationThreshold = 40000
+    optimizeAnimationsForMotionBlur = True
+    writeMaterials = True
+    useBaseTessellation = True
+
+    def process(self, instance):
+        staging_dir = self.staging_dir(instance)
```

Comment:
Ensure `gpuCache` plug-in is loaded.
```suggestion
        cmds.loadPlugin("gpuCache", quiet=True)

        staging_dir = self.staging_dir(instance)
```